fault/actions.py

```diff
@@ -34,6 +34,24 @@ def __init__(self, port, value):
         super().__init__(port, value)
 
 
+class Peek(Action):
```

Comment:
Wondering if we can rename this to like `Print(format_str, *args)` where we have some special syntax for a port (like `%v` or something else that's not used). The target functionality is the same, it just makes it more clear that this is just a print statement, not returning you the value inline as a `peek` normally would.

I.e. `val = peek(foo.O)` doesn't make sense here, though that's the normal usage of peek. Then if we have a non-compiled tester in the future that's running in real time, we can actually have a `peek` that returns a value.

Reply:
Okay that makes sense.